plasma_framework/contracts/src/vaults/EthVault.sol

```diff
@@ -24,15 +17,10 @@ contract EthVault is Vault {
 
         _validateDepositFormat(decodedTx);
 
-        bytes32 root = keccak256(_depositTx);
-        for (uint i = 0; i < 16; i++) {
-            root = keccak256(abi.encodePacked(root, zeroHashes[i]));
-        }
-
-        blockController.submitDepositBlock(root);
+        super._submitDepositBlock(_depositTx);
     }
 
-    function _validateDepositFormat(DepositTx.Transaction memory _deposit) private {
+    function _validateDepositFormat(DepositTx.Transaction memory _deposit) internal view {
```

Comment:
minor: why do we move to `internal`? is this something accidentally changed?

Reply:
Yeah, that was accidental, I'll change it back.
This method will disappear anyway...